makefiles/multislot.mk

```diff
@@ -0,0 +1,44 @@
+ifneq (,$(filter mcuboot, $(MAKECMDGOALS)))
+IMGTOOL ?= $(RIOTBASE)/dist/tools/mcuboot/imgtool.py
+override IMGTOOL := $(abspath $(IMGTOOL))
+
+ifndef SLOT0_SIZE
+$(error Board $(BOARD) does not define multislot parameters!)
+endif
+
+ifeq (nrf52dk, $(BOARD))
```

Comment:
please introduce a generally usable variable for this. This must be doable without board specifics int this file.
I'd suggest "FLASH_OFFSET" or sth similar, which would need to be exported, and used by our flashers (jlink, edbg, openocd)